patches/api/0225-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch

```diff
@@ -18,9 +27,21 @@ index 52290c43d1c02785c4cae4a73494a75cdc369e02..1efc97d88c38863bcd6cd4c11c8b88a1
 +     * Return the translation key for the Material, so the client can translate it into the active
 +     * locale when using a TranslatableComponent.
 +     * @return the translation key
++     * @deprecated use {@link #translationKey()}
 +     */
 +    @NotNull
++    @Deprecated
 +    public String getTranslationKey() {
++        return this.translationKey();
++    }
++
++    /**
++     * Return the translation key for the Material, so the client can translate it into the active
++     * locale when using a {@link net.kyori.adventure.text.TranslatableComponent}.
```

Comment:
Is there a reason why you're adding javadoc for these methods anyway?

Reply:
I guess it doesnt need one really. The doc on the interface probably suffices